x/concentrated-liquidity/lp.go

```diff
@@ -187,6 +192,10 @@ func (k Keeper) updatePosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddr
 		return sdk.Int{}, sdk.Int{}, err
 	}
 
+	if err := k.initOrUpdateFeeAccumulatorPosition(ctx, poolId, owner, liquidityDelta, lowerTick, upperTick); err != nil {
```

Comment:
I think we should keep this function to only be concerned with updates since it is called from both `createPosition` and `withdrawPosition`.

The "init" semantics here also feels redundant given that we initialize a position in `createPool` specifically anyways

Reply:
yeah, my first intention for this addition was to prevent erroring upon updating position wihtout having the fee accumulator initialized. But on the second thought, I agree with you that it is the correct behavior to error when fee accumulator position has not initialized